### ISSUE TITLE
Selecting a subset of Rfam families for annotation

### ISSUE THREAD

**Issue by AntonPetrov** [Rfam/rfam-website#2]
In some cases it makes sense to annotate sequences with a subset of Rfam families. For example, someone might want to select only Archaeal families to annotate Archaeal genomes.

@nawrockie in his [book chapter](http://link.springer.com/protocol/10.1007%2F978-1-62703-709-9_9) about annotating functional RNAs with [Infernal](http://infernal.janelia.org/) recommends to:
1. use [Rfam Taxonomy search](http://rfam.xfam.org/search?tab=searchTaxBlock#tabview=tab3) to search for a taxon of interest (for example, [Archaea](http://rfam.xfam.org/search/taxonomy?q=archaea))
2. save a list of identified families to a text file
3. download all covariance models from the Rfam FTP site (file *Rfam.cm.gz* *ftp://ftp.ebi.ac.uk/pub/databases/Rfam/CURRENT*)
4. extract families of interest using a custom script or **cmfetch**, a program from the Infernal suite. 

More details can be found in section [Using Infernal to Annotate Structural RNAs in an Archaeal genome](http://link.springer.com/protocol/10.1007%2F978-1-62703-709-9_9) of Eric's book chapter and in the [Infernal User Guide](http://selab.janelia.org/software/infernal/Userguide.pdf).

There are two problems:
* taxonomy search currently fails for some queries (e.g. when searching for Bacteria)
* there might be a more user-friendly way of performing this task.

I created this issue because many users send questions about this topic to the Rfam helpdesk, and this issue can be updated to reflect any new developments.

**Comment by nawrockie** [comment]
Good idea Anton. Also, you might consider precomputing at least 3 sets of models: Rfam.archaea.cm, Rfam.bacteria.cm, and Rfam.eukarya.cm (which would be reproducibly creatable using whatever method you eventually settle on) so that users could just directly download those. I think that would lower the barrier for users doing these targeted analyses (which are a good idea if you're really only interested in one domain or another -- to avoid issues with false positives). Also I think this would make it cleaner/easier for users to publish what they did (e.g. "I downloaded Rfam.X.cm and ran cmsearch", as opposed to: "I performed the following query on the Rfam taxonomy website, then saved that list of names, then used cmfetch...."). I'm not sure how Rob feels about this idea though since I don't believe they have analogous HMM files in Pfam...

**Comment by nawrockie** [comment]
And Rfam.virus.cm would probably be worth creating too.

**Comment by AntonPetrov** [comment]
Thanks Eric! I agree, precomputing sets of models could be a way to go. 

I think that we could also update the Rfam search so that one can filter the data using facets like in RNAcentral and then download a set of matching models. Then people can say: I did a search with this URL and downloaded N models.

**Comment by nawrockie** [comment]
That's a good idea too. Let me know if there's anything I can do to help.

**Comment by AntonPetrov** [comment]
Thanks Eric, will do!
